Tasks/VsTest/outputstream.ts

```diff
@@ -0,0 +1,33 @@
+import * as stream from 'stream';
+import * as os from 'os';
+import * as tl from 'vsts-task-lib/task';
+
+export class StringErrorWritable extends stream.Writable {
+    private value: string = '';
+
+    constructor(options: any) {
+        super(options);
+    }
+
+    _write(data: any, encoding: string, callback: Function): void {
+        this.value += data;
+
+        let errorString: string = data.toString();
+        let n = errorString.indexOf(os.EOL);
```

Comment:
too much of logic. 

let lines = errorString.split(os.EOL);
lines.forEach(function (line: string) {
  tl.error(line)
}